src/tap/line.py

```diff
@@ -111,7 +111,17 @@ def __str__(self):
         diagnostics = ""
         if self.diagnostics is not None:
             diagnostics = "\n" + self.diagnostics.rstrip()
-        return f"{is_not}ok {self.number} {self.description}{directive}{diagnostics}"
+        yaml_block = ""
+        if self._yaml_block is not None:
+            indent = " "
```

Comment:
I think the `indent` variable is making this look more complicated than it is. Can we remove the variable and just replace it with the space in the f-strings? I know that means that we'd have to modify more spots if the indent was ever changed, but, truthfully, I'm doubtful that the indent would ever change after this work is merged.